### DISCUSSION TITLE
[Feature suggestion] Add script toggle to the app

### DISCUSSION THREAD

**Discussion by boredwz** [AutoDarkMode/Windows-Auto-Night-Mode#688] [Development]
That would be really great.
You can also add a "description" parameter to briefly tell what the script is about.

![image](https://user-images.githubusercontent.com/87862400/220162910-755dbf6b-4368-4629-af7a-0245f19a9310.png)
![image](https://user-images.githubusercontent.com/87862400/220163218-b8bf5c37-931d-4d9b-af81-941865c48afe.png)

**Comment by Spiritreader** [comment]
That's a neat idea. Will look into it if I ever get around to continue working on scripts

**Comment by Jay-o-Way** [comment]


**Reply by Jay-o-Way** [reply]
Oh wait, I see now that is not true. The idea is to have an on/off switch for each script, but we dont have that. Sorry.

**Comment by ReuelAlbert-Dev** [comment]
Allow users to enable or disable individual scripts without removing them entirely—making script management more flexible and user-friendly.
